neqo-transport/src/connection.rs

```diff
@@ -600,15 +600,14 @@ impl Connection {
     }
 
     /// Set or clear the qlog for this connection.
-    pub fn set_qlog(&mut self, qlog: Option<NeqoQlog>) {
-        let conn_ql = Rc::new(RefCell::new(qlog));
-        self.loss_recovery.set_qlog(conn_ql.clone());
-        self.qlog = conn_ql;
+    pub fn set_qlog(&mut self, qlog: &NeqoQlog) {
```

Comment:
As above, though this will need to clone for passing to loss_recovery

Reply:
done